<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogReport</class>
 <widget class="QDialog" name="DialogReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>711</width>
     <height>62</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="minimumSize">
       <size>
        <width>211</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>211</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>ЗАПИС НА USB ПАМЕТ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnShow">
      <property name="minimumSize">
       <size>
        <width>211</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>211</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>ПОКАЖИ</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnExit">
      <property name="minimumSize">
       <size>
        <width>211</width>
        <height>60</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>211</width>
        <height>60</height>
       </size>
      </property>
      <property name="text">
       <string>ИЗХОД</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>781</width>
     <height>391</height>
    </rect>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStartTimeUp">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>600</number>
   </property>
   <property name="autoRepeatInterval">
    <number>300</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEndTimeUp">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>290</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>600</number>
   </property>
   <property name="autoRepeatInterval">
    <number>300</number>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidgetEnd">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>30</y>
     <width>361</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblStart">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>НАЧАЛО</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditEnd">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>290</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>80</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>65</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="time">
    <time>
     <hour>23</hour>
     <minute>59</minute>
     <second>0</second>
    </time>
   </property>
  </widget>
  <widget class="QLabel" name="lblEnd">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>КРАЙ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEndTimeDown">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>600</number>
   </property>
   <property name="autoRepeatInterval">
    <number>300</number>
   </property>
  </widget>
  <widget class="QTimeEdit" name="timeEditStart">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>80</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>65</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStartTimeDown">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>600</number>
   </property>
   <property name="autoRepeatInterval">
    <number>300</number>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidgetStart">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>361</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
